assets/shaders/viewport_render.wgsl

```diff
@@ -292,7 +292,7 @@ fn get_node_occupancy_bitmap(sized_node_meta: u32) -> u32 {
 }
 
 // Functionality-wise this function is more generic, than its coutnerpart
```

Comment:
Maybe we need tom top-level docs to explain what a brick is? I mean, there is so much jargon flying around... if you dive deep, you need to read papers, for sure, but generally, we could try to get some basic explanations in.

Is that something we want?

Reply:
Oh yes definitely, will include it in this PR